client/stylesheets/_toCat.scss

```diff
@@ -1028,12 +1028,17 @@ HELP | ABOUT | FAQ | SAFETY | PRIVACY | CONTACT **/
 
 .helplogo {
     color: $color-spacegrey;
+    font-weight: 400 !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces